cmake/public/LoadHIP.cmake

```diff
@@ -96,8 +103,23 @@ set(CMAKE_MODULE_PATH ${HIP_PATH}/cmake ${CMAKE_MODULE_PATH})
 # Disable Asserts In Code (Can't use asserts on HIP stack.)
 ADD_DEFINITIONS(-DNDEBUG)
 
+macro(find_package_and_print_version PACKAGE_NAME)
+  find_package("${PACKAGE_NAME}" ${ARGN})
+  message("${PACKAGE_NAME} VERSION: ${${PACKAGE_NAME}_VERSION}")
+endmacro()
+
+message("\n***** Library versions from dpkg *****\n")
+execute_process(COMMAND dpkg -l COMMAND grep rocm-dev COMMAND awk "{print $2 \" VERSION: \" $3}")
```

Comment:
These need to be put inside the ""IF(HIP_FOUND)" clause below, otherwise they will be run in all builds.

Reply:
Can do, I was under the impression `LoadHIP.cmake` was only triggered if `USE_ROCM=1`